test_ngi_config.yaml

```diff
@@ -0,0 +1,150 @@
+    flowcell_inbox: 
+            - /proj/a2014205/archive
+            - /proj/a2015179/archive
+
+logging:
+    # the log file itself is compulsory to be defined, or you will get a nasty exception
+    # to make sure you are defining it, and not overwriting the production one below, it is left commented out
+    #log_file: /proj/a2014205/ngi_resources/ngi_pipeline.log
+
+paths: # Hard code paths here if you are that kind of a person
+    binaries:
+        #bowtie2:
+        #fastqc:
+        #fastq_screen:
+    references:
+
+    #log: /proj/a2010002/data/log
+    #store_dir: /proj/a2010002/archive
+
+piper:
+    # The engine we are usually runnig - further engines should be configured also like this.
+    # Also can be set as an environmental variable $PIPER_QSCRIPTS_DIR
+    path_to_piper_qscripts: /proj/a2014205/software/piper/qscripts
+    load_modules:
+        - java/sun_jdk1.7.0_25
+        - R/2.15.0
+    threads: 16
+    job_walltime:
+        merge_process_variantcall: "10-00:00:00"
+    #sample:
+    #    required_autosomal_coverage: 28.4
+    shell_jobrunner: Shell
+    #shell_jobrunner: ParallelShell --super_charge --ways_to_split 4
+    #jobNative:
+    #    - arg1
+    #    - arg2
+    #    - arg3
+
+qc:
+    # These qc modules are related to pre-analysis QC runs
+    load_modules:
+        - bioinfo-tools
+    fastqc:
+        load_modules:
+            - FastQC
+        threads: 16
+    fastq_screen:
+        config_path: "/proj/a2014205/ngi_resources/fastq_screen.nestor.conf"
+        load_modules:
+            - bowtie2
+            - fastq_screen
+        subsample_reads: 200000
+        threads: 1
+
+slurm:
+    extra_params:
+        "--qos": "seqver"
+    cores: 16
+
+supported_genomes:
+    #"GRCh37": "/apus/data/uppnex/reference/Homo_sapiens/GRCh37/concat/Homo_sapiens.GRCh37.57.dna.concat.fa"
+    "GRCh37": /proj/a2014205/piper_references/gatk_bundle/2.8/b37/human_g1k_v37.fasta
+    "GRCm38": "/apus/data/uppnex/reference/Mus_musculus/GRCm38/concat/Mus_musculus.GRCm38.69.dna.concat.fa"
+    "rn4": None
+    "saccer2": None
+    "dm3": None
+    "tair9": None
+    "xentro2": None
+    "ws210": None
+    "canfam3": None
+
+#project:
+#    INBOX: /proj/a2010002/archive
+
+test_data:
+    workflows:
+        whole_genome_reseq:
+            test_project:
+                project_id: P0000
+                project_name: "Y.Mom_15_01"
+                bpa: "whole_genome_reseq"
+            customize_config:
+                analysis:
+                    top_dir: /proj/a2014205/nobackup/NGI/test_data/
+                charon:
+                    charon_base_url: http://charon-dev.scilifelab.se
+            local_files:
+                flowcell: /proj/a2014205/nobackup/NGI/test_data/whole_genome_reseq/150424_ST-E00214_0031_BH2WY7CCXX
+                #project: /proj/a2014205/nobackup/NGI/test_data/whole_genome_reseq/DATA/P0000
+                vcf: /proj/a2014205/nobackup/NGI/test_data/whole_genome_reseq/vcf/
+
+mail:
```

Comment:
this should be added to the nestor_ngi_config.yaml, and here the mail should be changed with a mock one `user@facility.se`